config.js

```diff
@@ -37,12 +43,12 @@ module.exports = {
       freeSocketTimeout: 30000,
     },
     apiClientId: get('API_CLIENT_ID', 'my-diary', requiredInProduction),
-    apiClientSecret: get('API_CLIENT_SECRET', 'clientsecret', requiredInProduction),
+    apiClientSecret: get('API_CLIENT_SECRET', 'kJwGCYAhn4AYRQF9', requiredInProduction),
   },
   app: {
     production,
     mailTo: process.env.MAIL_TO || 'feedback@digital.justice.gov.uk',
-    url: process.env.CHECK_MY_DIARY_URL || `http://localhost:${process.env.PORT || 3005}`,
+    url: process.env.CHECK_MY_DIARY_URL || 'http://localhost:3000',
```

Comment:
This is a bit messy, but it's defaulting to the testing port if no env `PORT` is set